src/theme/material/list-item.m.css

```diff
@@ -1,14 +1,23 @@
 .root {
-	composes: mdc-list-item from '@material/list/dist/mdc.list.css';
 	color: var(--mdc-text-color);
 	overflow: hidden;
 	text-overflow: ellipsis;
 	white-space: nowrap;
-}
-
-.root.height {
 	min-height: 48px;
 	height: 100%;
+	composes: mdc-list-item from '@material/list/dist/mdc.list.css';
+}
+
+.smallPadding {
+}
+
+.mediumPadding {
+	padding-left: calc(var(--mdc-theme-grid-base) * 2);
+	padding-right: calc(var(--mdc-theme-grid-base) * 2);
+}
+
+.noPadding {
```

Comment:
Can you avoid `!important` by adding `.root` to the selector?

Reply:
most likely, could be that material css is using a gnarly selector though